### PR TITLE
Updated Models to better include relationships. Created a composite key for accountHolder. added queries.

### PR DESCRIPTION
**Pull request by Joshua-Mallory**

### CODE REVIEW

File: src/main/java/com/aline/core/model/account/Member.java [hunk 1]
```diff
@@ -18,6 +25,10 @@
...
 	@Id
 	@GeneratedValue(strategy = GenerationType.IDENTITY)
-	Long id;
-	Long user_id;
+	private Long id;
+	@OneToOne(optional = false)
+	@JoinColumn(name = "user_id")
+	private User user;
+	@OneToMany(mappedBy = "member")
+	private List<AccountHolder> accountHold;
```
Comment (leandroyabut) [inline src/main/java/com/aline/core/model/account/Member.java:33]: This list should be `List<Account> accounts`

File: src/main/java/com/aline/core/model/account/AccountHolder.java [hunk 2]
```diff
@@ -14,8 +17,18 @@
...
+	@MapsId("member_id")
+	@JoinColumn(name = "member_id")
+	private Member member;
+	@ManyToOne
+	@MapsId("account_id")
+	@JoinColumn(name = "account_id")
+	private Account account;
+	@ManyToOne(optional = false)
+	@JoinColumn(name = "account_holder_type_id")
+	private AccountHolderType accountHolderType;
```
Comment (leandroyabut) [inline src/main/java/com/aline/core/model/account/AccountHolder.java:33]: This should be `List<Members> members`

**Review by leandroyabut — DISMISSED** [review]
Everything looks awesome! If you could just change the 2 lines that I commented that would be awesome. This looks great Josh.

File: src/main/java/com/aline/core/model/account/AccountHolder.java (hunk 2 above)
Comment (beki01) [reply src/main/java/com/aline/core/model/account/AccountHolder.java:33]: This will work for now. Approving changing.

**Review by beki01 — APPROVED** [review]

**Pull request merged**